numpy/core/src/umath/loops.c.src

```diff
@@ -865,6 +868,61 @@ NPY_NO_EXPORT void
         }
     }
 }
+#else
+NPY_NO_EXPORT void
+@TYPE@_divide(char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
+{
+    BINARY_LOOP_BASE
+
+    if(!is2) {
+        struct libdivide_@div@_t fast_d = libdivide_@div@_gen(*(int*)ip2);
+        const @type@ in2 = *(@type@ *)ip2;
```

Comment:
This case looks very unsafe to me. At a minimum this should be
```suggestion
        const @type@ in2 = *(@type@ *)ip2;
        struct libdivide_@div@_t fast_d = libdivide_@div@_gen((int)in2);
```
but even then the cast to `int` is dangerous for division on int64

Reply:
Oh yeah, my bad. The cast itself is not needed, was using ip2 before, and forgot to switch.